manifests/params.pp

```diff
@@ -340,7 +340,7 @@
       $config_file         = '/etc/my.cnf'
       $includedir          = undef
       $datadir             = '/var/mysql'
-      $log_error           = "/var/mysql/${::hostname}.err"
+      $log_error           = "/var/mysql/${facts['os']['hostname']}.err"
```

Comment:
```suggestion
      $log_error           = "/var/mysql/${facts['networking']['hostname']}.err"
```

Reply:
Investigated this and confirmed a number of facts hidden in the test files that were not caught by our Lint plugin. Will amend soon.

Reply:
That should be those addressed